src/libm/sleefqp.c

```diff
@@ -22,10 +22,6 @@
 #pragma fp_contract (off)
 #endif
 
-#if defined(__GNUC__)
```

Comment:
Could you explain why you removed this?

Reply:
because it was failing compiling as it is already define in misc.h. So redefinition occured...

Reply:
I see. You are very quick!